Gruntfile.js

```diff
@@ -85,8 +85,51 @@ module.exports = function(grunt) {
             }
         },
         clean: {
+            coverage: ['coverage/', 'coverage.lcov', '.nyc_output/'],
             dist: ['.dist/'],
+            packages: ['Consus-Client-*/'],
             test: ['.test/']
+        },
+        electron: {
+            win32Build: {
+                options: {
+                    name: 'Consus-Client',
+                    dir: '.',
+                    out: '.',
+                    version: '1.4.2',
+                    platform: 'win32',
+                    arch: 'x64',
+                    'no-prune': true,
```

Comment:
Is this a real option, I didn't see it [sindresorhus/grunt-electron](github.com/sindresorhus/grunt-electron) nor [electron-userland/electron-packager](github.com/electron-userland/electron-packager). I *did*, however, find the lovely [`prune`](https://github.com/electron-userland/electron-packager/blob/master/docs/api.md#prune) option, which I think we may want to reduce the final package size.